contributing.md

```diff
@@ -289,6 +289,66 @@ ansible-playbook --skip-tags tests::cleanup \
     -i testhost, tests/playbooks/tests_802_1x.yml
 ```
 
+### Integration tests with podman
+1. Create `~/.ansible/collections/ansible_collections/containers/podman/` if this
+directory does not exist and `cd` into this directory.
+```bash
+mkdir -p ~/.ansible/collections/ansible_collections/containers/podman/
+cd ~/.ansible/collections/ansible_collections/containers/podman/
+```
+
+2. Clone the collection plugins for Ansible-Podman into the current directory.
+```bash
+git clone https://github.com/containers/ansible-podman-collections.git .
+```
+
+3. Change directory into the `network/tests`, `network` is the Repo you cloned from
+your local fork.
+```bash
+cd ~/network/tests
+```
+
+
+4. Use podman with `-d` to run in the background (daemon). Use `c7` because
+`centos/systemd` is centos7.
+```bash
+podman run --name c7 --rm --privileged -v /sys/fs/cgroup:/sys/fs/cgroup:ro -d registry.centos.org/centos/systemd
+```
+
+5. Use `podman unshare` first to run "podman mount" in root mode, use `-vi` to run
+ansible as inventory in verbose mode, use `-c podman` to use collection plugins. Note,
+the failed test files are:
+- `tests_802_1x_nm.yml` 
+- `tests_802_1x_updated_nm.yml`
+- `tests_bond_initscripts.yml`
+- `tests_bond_nm.yml`
+- `tests_bridge_initscripts.yml`
+- `tests_bridge_nm.yml`
+- `tests_default_nm.yml`
+- `tests_ethernet_nm.yml`
+- `tests_reapply_nm.yml`
+- `tests_states_initscripts.yml`
+- `tests_states_nm.yml`
```

Comment:
What's the failure for these?

Reply:
included in above comments.

Reply:
@tyll Do you want the logs into the documentation? I think it will be confusing for the users..

Reply:
no need to add the actual errors to the documentation. I was just confused that some basic appeared not to be working and needed to take a look at the logs.